packages/blade/src/components/Popover/_decisions/decisions.md

```diff
@@ -0,0 +1,275 @@
+# Popover API Decisions <!-- omit in toc -->
+
+The popover component is used to provide context as well as enable users to take certain actions on it. They are designed to display relevant information without requiring the user to navigate to a different page or dialog. Popovers are triggered on click or onload.
+
+<img src="./assets/popover-thumbnail.png" width="380" alt="Popover Thumbnail" />
+
+- [Design](#design)
+- [Basic Features](#basic-features)
```

Comment:
add before this section

* Link to Tooltip decision
* Comparision should come here before even we start popover related things?

Reply:
Can you also add comparision to tooltip decisions?

Reply:
> Link to Tooltip decision

Added

> Comparision should come here before even we start popover related things?

Might get a bit overwhelming at first, let them go through the basic functionality first and then they will eventually reach the comparison.